From cb1c0cc7d31d21430a95fc5c5b40c8375ffab299 Mon Sep 17 00:00:00 2001
From: Petr Oros <poros@redhat.com>
Date: Wed, 17 Sep 2014 16:25:12 -0400
Subject: [redhat] kabi: Adding symbol seq_read (fs/seq_file.c)

Message-id: <997f5c9f7c1ce9f3aeda1ad99293e8a2201de247.1410510773.git.poros@redhat.com>
Patchwork-id: 96069
O-Subject: [PATCH RHEL7.1 192/233] kabi: Adding symbol seq_read (fs/seq_file.c)
Bugzilla: 1062361 1071099
RH-Acked-by: Alexander Viro <aviro@redhat.com>

Bugzilla: 1071099,1062361
    https://bugzilla.redhat.com/show_bug.cgi?id=1071099
    https://bugzilla.redhat.com/show_bug.cgi?id=1062361

    Description:
    ============
    The function is used as the .read method for the read operation on proc interface.

Signed-off-by: Petr Oros <poros@redhat.com>
---
 create mode 100644 redhat/kabi/kabi-module/kabi_ppc64/seq_read
 create mode 100644 redhat/kabi/kabi-module/kabi_s390x/seq_read
 create mode 100644 redhat/kabi/kabi-module/kabi_x86_64/seq_read

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/kabi/kabi-module/kabi_ppc64/seq_read  b/redhat/kabi/kabi-module/kabi_ppc64/seq_read 
new file mode 100644
index 0000000..6695fbb
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_ppc64/seq_read  
@@ -0,0 +1,2 @@
+#1-
+0xd09c13c0 seq_read vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_s390x/seq_read  b/redhat/kabi/kabi-module/kabi_s390x/seq_read 
new file mode 100644
index 0000000..d88d849
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_s390x/seq_read  
@@ -0,0 +1,2 @@
+#1-
+0xbc4941ce seq_read vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_x86_64/seq_read  b/redhat/kabi/kabi-module/kabi_x86_64/seq_read 
new file mode 100644
index 0000000..5d41b27
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_x86_64/seq_read  
@@ -0,0 +1,2 @@
+#1-
+0x9c3df9b4 seq_read vmlinux EXPORT_SYMBOL
-- 
1.7.1